bin/cfg/TR1X_gameflow.json5

```diff
@@ -1,97 +1,40 @@
 {
     // NOTE: bad changes to this file may result in crashes.
     // Lines starting with double slashes are comments and are ignored.
+    // Refer to https://github.com/LostArtefacts/TR1X/GAMEFLOW.md for usage.
 
-    // Path to the main menu background image.
     "main_menu_picture": "data/titleh.png",
-
-    // Path to the savegame file.
     "savegame_fmt_legacy": "saveati.%d",
     "savegame_fmt_bson": "save_tr1_%02d.dat",
-
-    // Forces game mode selection to be disabled if true so the user can't
-    // select NG+ modes until a playthrough is completed. Overrides the config
-    // option enable_game_modes.
     "force_disable_game_modes": false,
-
-    // Forces save crystals to be enabled if true. Overrides the config option
-    // enable_save_crystals.
     "force_enable_save_crystals": false,
-
-    // Seconds to pass in the main menu before playing the demo.
     "demo_delay": 16,
-
-    // Water color (R, G, B). 1.0 means pass-through, 0.0 means no value at all.
-    // [0.6, 0.7, 1.0] is original DOS version filter,
-    // [0.45, 1.0, 1.0] is default TombATI filter.
     "water_color": [0.45, 1.0, 1.0],
```

Comment:
Isn't this water color value actually different from TombATI since TR1X better matches the PS1 TR1? The wording makes it sound like we match TombATI, but I don't think that's the case. @rr-

Reply:
Looking at `tombati.ini` it's not clear what the blue value defaults to, but red and green are certainly as described here.

Reply:
Blue is always 0xFF in TombATI.